src/routes/(admin)/questionbank/single-question/[action]/[id]/page.server.test.ts

```
@@ -0,0 +1,782 @@
+				.mockResolvedValueOnce({ ok: true, json: async () => ({ id: '10', options: [] }) })
+				.mockResolvedValueOnce({ ok: true, json: async () => ({}) })
+				.mockResolvedValueOnce({ ok: true, json: async () => ({}) })
+				.mockResolvedValueOnce({ ok: true, json: async () => ({}) });
+
+			try {
+				await actions.save({
+					request: new Request('http://localhost', { method: 'POST' }),
+					params: { action: 'edit', id: '10' },
+					cookies: mockCookies
+				} as any);
+			} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)